test/e2e/perf/logging_soak.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package perf
```

Comment:
But this test is owned by logging team (similarly as Node benchmarks are owned by node team). I don't think we should move it here.